docs/tools/sample-nodeos-configs.md

```diff
@@ -2,6 +2,8 @@
 id: sample-nodeos-configs
 title: Sample Nodeos Configurations 
 sidebar_label: Sample Nodeos Configurations 
+description: Sample for Nodeos Configurations
+keywords: [nodeos, config, configuration, Nodeos Configuration, What is Nodeos, How to configure Nodeos?, How to configurate a nodo?, EOS, EOS Costa Rica]
```

Comment:
change "How to configurate a nodo"
* configure instead of configurate
* maybe nodeos instead of nodo

Reply:
Attended.